helmfile/values/fluentd-user.yaml.gotmpl

```diff
@@ -62,6 +62,19 @@ extraConfigMaps:
         </pattern>
       </parse>
     </source>
+    # Index the authlog
+    <source>
+      @id authlog
+      @type tail
+      path /var/log/auth.log
+      pos_file /var/log/auth.pos
+      tag authlog.*
```

Comment:
We could do without the `.*` if we want. All it does is add the add the dotnotation path, e.g. events will be emitted with tag `authlog.var.log.auth.log`.

Reply:
We seem to use dotnotation for the other indicies so I'm leaving it for now.